PrestaShop/autoupgrade
# Disable the update button when the php version is not compatible

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Disable the update button when the php version is not compatible , Add Waring when a local archive is used , Add image php compatible version 
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes PrestaShop/PrestaShop#25659  & Fixes PrestaShop/PrestaShop#25657
| How to test?      |  See below
| Possible impacts? | the module need an update for the next PrestaShop 


1. git clone `https://github.com/PrestaShop/PrestaShop.git --branch 1.6.1.24`  To have a possible update and a no comptaible php Version
2. Install the module.
3. Try to update with a non compatible php Version 
4. Use the local archive and see the warning
5. Use a compatible PHP version and check no error.

You can use a another version of PrestaShop

Acutaly i use a constant, @marionf say to use a chanel from PrestaShop Api but he don't exist.

When an xml file is available on https://api.prestashop.com/xml/XXXX I can use it.

![Capture d’écran de 2021-09-24 10-46-40](https://user-images.githubusercontent.com/5064590/134646330-3e8a9465-cd9b-4bca-88ee-4367ae77091b.png)
![Capture d’écran de 2021-09-24 10-43-51](https://user-images.githubusercontent.com/5064590/134646340-c3559f64-8377-471b-bbe1-663a3a468ade.png)
![Capture d’écran de 2021-09-24 10-44-02](https://user-images.githubusercontent.com/5064590/134646347-981158f1-1314-449e-89da-cdfdd9bc12f8.png)
